tests/transformers/test_transformers_model_export.py

```diff
@@ -1859,12 +1844,8 @@ def test_classifier_pipeline_pyfunc_predict(text_classification_pipeline, tmp_pa
     )
     values = PredictionsResponse.from_json(response.content.decode("utf-8")).get_predictions()
 
-    assert values.to_dict(orient="records") == [
-        {0: "NEGATIVE"},
-        {0: "NEGATIVE"},
-        {0: "POSITIVE"},
-        {0: "POSITIVE"},
-    ]
+    assert len(values.to_dict()) == 2
```

Comment:
all of these validations are being moved to structural relationships since scores are somewhat non-deterministic depending on the model used.